### ISSUE TITLE
Missing/wrong country flags

### ISSUE THREAD

**Issue by theSarrum** [multitheftauto/mtasa-resources#318]
**Which resource(s) have this problem?**
admin

**Describe the bug**
Not exactly a bug, but I found that some of the flags are missing and some that need to be removed/replaced.

<details>
 <summary>Flags that are missing:</summary>

```
BL - Saint Barthélemy
BQ - Bonaire, Sint Eustatius and Saba
CW - Curaçao
FK - Falkland Islands (the) [Malvinas]
FM - Micronesia (Federated States of)
GF - French Guiana
GG - Guernsey
GH - Ghana
GI - Gibraltar
GN - Guinea
GQ - Equatorial Guinea
GS - South Georgia and the South Sandwich Islands
GU - Guam
HM - Heard Island and McDonald Islands
IM - Isle of Man
IO - British Indian Ocean Territory (the)
KE - Kenya
KH - Cambodia
KI - Kiribati
KM - Comoros (the)
KY - Cayman Islands (the)
LA - Lao People's Democratic Republic (the)
LC - Saint Lucia
LI - Liechtenstein
LR - Liberia
LS - Lesotho
MF - Saint Martin (French part)
ML - Mali
MO - Macao
MP - Northern Mariana Islands (the)
MQ - Martinique
MR - Mauritania
MS - Montserrat
MW - Malawi
NA - Namibia
NE - Niger (the)
NF - Norfolk Island
NU - Niue
PF - French Polynesia
PN - Pitcairn
PW - Palau
SB - Solomon Islands
SH - Saint Helena, Ascension and Tristan da Cunha
SM - San Marino
SN - Senegal
SR - Suriname
SS - South Sudan
ST - Sao Tome and Principe
SX - Sint Maarten (Dutch part)
SZ - Eswatini
TC - Turks and Caicos Islands (the)
TF - French Southern Territories (the)
TG - Togo
TL - Timor-Leste
TM - Turkmenistan
TZ - Tanzania, the United Republic of
UG - Uganda
VA - Holy See (the)
VC - Saint Vincent and the Grenadines
VG - Virgin Islands (British)
VI - Virgin Islands (U.S.)
VU - Vanuatu
ZM - Zambia
ZW - Zimbabwe
```
</details>

Flags that need to be replaced:
`ad.png` - wrong Andorra flag
`us.png` - should use the same flag as `um.png`

Flags that need to be removed:
`ac.png`
`an.png`
`cs.png`
~~`eu.png`~~ - _might be worth removing, but it is used when a specific country code has not been designated_
`ja.png` - a duplicate of jp.png
`mil.png`
`uk.png` - a duplicate of gb.png
~~`xk.png`~~ - _a self assigned code (there is no ISO 3166 country code for Kosovo, but it is used in the csv)_
`yu.png`

**Comment by jlillis** [comment]
I think we should address this as part of #222

**Comment by Fernando-A-Rocha** [comment]
@theSarrum 
Thank you for the list. I ran some tests to make sure it's right and 10/10 on point. I'll see if I can add the missing flags & replace the wrong ones. I've already removed all the invalid flags in that PR above.

**Comment by jlillis** [comment]
Resolved via #405
